### PR TITLE
Add animal galleries, project paragraph to main page; Style changes to images and links

### PR DESCRIPTION
**Pull request by Corramel**

### CODE REVIEW

**Comment by Corramel** [comment]
Forgot to add some files.

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,18 +1,46 @@
 body {
-    background-color: seashell;
+    background-color: rgb(255, 245, 238);
 }
 
 .pImg {
-    max-width:10%;
+    max-width:5%;
```
Comment (matty377) [inline portfolio/src/main/webapp/style.css:6]: Why percent and not an explicit value in pixels? or something like em or rem which are relative to font size. Just curious, no need to change.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
+
 .splash {
     max-width:50%;
     max-height:auto;
     padding: 10px;
 }
 
+#navExternal {
+    float:right;
+
```
Comment (matty377) [inline portfolio/src/main/webapp/style.css:26]: nitpick - extra line

**Review by matty377 — COMMENTED** [review]
LGTM. Nice use of CSS pseudoselectors for the hover and visited link behaviors.

**Pull request closed without merging**